app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -57,18 +58,21 @@ class ProfileChooserFragmentPresenter @Inject constructor(
   private val viewModelProvider: ViewModelProvider<ProfileChooserViewModel>,
   private val profileManagementController: ProfileManagementController
 ) {
+  private lateinit var binding: ProfileChooserFragmentBinding
+
   private val chooserViewModel: ProfileChooserViewModel by lazy {
     getProfileChooserViewModel()
   }
 
   /** Binds ViewModel and sets up RecyclerView Adapter. */
   fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?): View? {
-    val binding =
+    binding =
       ProfileChooserFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)
```

Comment:
Convert this to single line:
`binding = ProfileChooserFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)`

Reply:
Done.